docs/source/apps/cs2cs.rst

```diff
@@ -261,7 +261,7 @@ The following script
 ::
 
     cs2cs +proj=latlong +datum=NAD83 +to +proj=utm +zone=10 +datum=NAD27 -r <<EOF
-    45d15'33.1" 111.5W
+    45°15'33.1" 111.5W
```

Comment:
I'd prefer to keep this with `d`, as this is used to generate man pages, which are generally plain-text.

Reply:
Nothing wrong with UTF8 in man pages; in fact there are several in the non-English pages in `/usr/share/man/{anything but en}`.

Reply:
@mwtoews I can put it back to `d` if it helps with man page generation, but then there would be nothing at all in the docs to clue users in to the fact that they can use the degree sign.  I feel like it ought to be mentioned *somewhere*, and the cs2cs page was the only place I found that touches on the DMS string format.  I'm open to other ideas about how to document the feature.

Reply:
Ideally it would be described in the online docs, but there is no perfect place for it. This feature could be reworked in [quickstart](https://proj.org/usage/quickstart.html) or in one of the following pages, but that's beyond what's expected here. So sure, let's leave it here for now.